portal-ui/src/screens/Console/Logs/ErrorLogs/LogLine.tsx

```diff
@@ -62,46 +62,55 @@ const messageForConsoleMsg = (log: LogMessage) => {
   );
 };
 const messageForError = (log: LogMessage) => {
-  const dataStyle = { color: "green" };
+  const dataStyle = {
+    color: "#C83B51",
+    fontWeight: 400,
+    fontFamily: "'monospace'",
```

Comment:
Please remove single quotes, this leads to fallback font

```suggestion
    fontFamily: "monospace",
```
<img width="1159" alt="Screen Shot 2022-03-17 at 12 52 50" src="https://user-images.githubusercontent.com/33497058/158875552-c1243053-ccb0-4d8f-8738-b0b6e188a2d3.png">